.travis.yml

```diff
@@ -52,3 +52,8 @@ jobs:
       env: STATIC_ANALYSIS
       install: travis_retry composer install --prefer-dist
       script: vendor/bin/psalm
+
+    - stage: Backwards Compatibility
+      env: STATIC_ANALYSIS
+      install: travis_retry composer install --prefer-dist
+      script: vendor/bin/roave-backward-compatibility-check --from=v1.5.0
```

Comment:
Hardcoding tag name here is not a good idea, can we use some kind of wildcard, like _latest 1.x tag_?